src/Markdown.MAML/Renderer/Markdownv2Renderer.cs

```diff
@@ -73,18 +73,18 @@ private string MamlModelToString(MamlCommand mamlCommand, Hashtable yamlHeader,
                     RenderCleaner.NormalizeQuotesAndDashes(
                         _stringBuilder.ToString())));
         }
-        
+
```

Comment:
Please, don't mix style/whitespaces changes in the same commits and PRs and the actual code changes in the future. It makes it much harder to review.